packages/node/src/lib/novu.ts

```diff
@@ -39,37 +58,139 @@ export class Novu extends EventEmitter {
       },
     });
 
-    this.subscribers = new Subscribers(this.http);
-    this.environments = new Environments(this.http);
-    this.events = new Events(this.http);
-    this.trigger = this.events.trigger;
-    this.bulkTrigger = this.events.bulkTrigger;
-    this.broadcast = this.events.broadcast;
-    this.changes = new Changes(this.http);
-    this.layouts = new Layouts(this.http);
-    this.notificationGroups = new NotificationGroups(this.http);
-    this.notificationTemplates = new NotificationTemplates(this.http);
-    this.feeds = new Feeds(this.http);
-    this.topics = new Topics(this.http);
-    this.integrations = new Integrations(this.http);
-    this.messages = new Messages(this.http);
+    this.subscribers = new Subscribers(this);
+    this.environments = new Environments(this);
+    this.events = new Events(this);
+    this.changes = new Changes(this);
+    this.layouts = new Layouts(this);
+    this.notificationGroups = new NotificationGroups(this);
+    this.notificationTemplates = new NotificationTemplates(this);
+    this.workflows = new Workflows(this);
+    this.feeds = new Feeds(this);
+    this.topics = new Topics(this);
+    this.integrations = new Integrations(this);
+    this.messages = new Messages(this);
   }
 
-  public trigger: typeof Events.prototype.trigger;
-
-  public bulkTrigger: typeof Events.prototype.bulkTrigger;
+  public trigger(workflowIdentifier: string, data: ITriggerPayloadOptions) {
+    return this.events.trigger(workflowIdentifier, data);
+  }
 
-  public broadcast: typeof Events.prototype.broadcast;
+  public bulkTrigger(events: IBulkEvents[]) {
+    return this.events.bulkTrigger(events);
+  }
+  public broadcast(workflowIdentifier: string, data: IBroadcastPayloadOptions) {
+    return this.events.broadcast(workflowIdentifier, data);
+  }
 
   private buildBackendUrl(config?: INovuConfiguration) {
-    const novuVersion = 'v1';
+    const novuApiVersion = 'v1';
 
     if (!config?.backendUrl) {
-      return `https://api.novu.co/${novuVersion}`;
+      return `https://api.novu.co/${novuApiVersion}`;
     }
 
     return config?.backendUrl.includes('novu.co/v')
       ? config?.backendUrl
-      : config?.backendUrl + `/${novuVersion}`;
+      : config?.backendUrl + `/${novuApiVersion}`;
+  }
+
+  async get(
+    path: string,
+    params?: Record<string, unknown>
+  ): Promise<AxiosResponse> {
+    try {
+      if (params) {
+        return await this.http.get(path, params);
+      } else {
+        return await this.http.get(path);
+      }
+    } catch (error: unknown) {
+      this.handleError(error);
+      throw error;
+    }
+  }
```

Comment:
I am not sure I like that it is here. Why are we changing it? Because of the errors?

Reply:
Yes, added custom axios methods to return only specific error information
Earlier our SDK was returning axios error. It was hard to debug error information from that error.

Reply:
Removed Error handling as per Pawel's suggestion
Reverted back these changes